### DISCUSSION TITLE
Version stable

### DISCUSSION THREAD

**Discussion by elmerdu69** [OUARZA/Propluvia#50] [Ideas]
Hello,
Après tous ces updates et bugfixs sur la version beta, on a un plugin qui est maintenant bien fonctionnel. Bon ok avec peu de retours...
Je pense maintenant qu'il mérite de passer en sable.  Ou bien vois tu encore des choses à améliorer?

Dans tous les cas avant de passer en stable je ferai un dernier PR pour nettoyer le code et commandes inutiles.  
Le dernier bug que j'ai relevé, je le corrigerai un peu plus tard.
tu es ok avec ça ?

**Comment by OUARZA** [comment]
Hello,
Ça me va.
On en profiterait pas pour l'inscrire dans le market de Jeedom?

**Reply by elmerdu69** [reply]
Oui c'est l'occasion de le faire aussi.
Tu t'en occupes ?
Je ne sais pas ce qu'il y'a à faire

**Reply by OUARZA** [reply]
Oui, je vais le faire, j'ai trouvé un tuto, c'est un peu pénible à faire, mais je me dis que c'est l'occasion de l'ouvrir à plus de monde

**Reply by OUARZA** [reply]
@elmerdu69 tu préfères le faire ou je le fais ?
https://doc.jeedom.com/fr_FR/dev/publication_plugin

Le seul truc qui me dérange un peu c'est d'être identifié comme étant développeur alors que je ne le suis pas et que tu as tout fait :)

**Reply by elmerdu69** [reply]
À vrai dire peu importe pour moi. Oui j'ai fait une partie du code, mais l'idée vient de toi. Et puis tu t'occupes de la doc et de la com sur le forum. 
Donc si tu préfères que je le fasses, je le fais. Mais ne t'en prive pour autant.

**Reply by OUARZA** [reply]
Yo, veux-tu le rendre payant ?
Bien-sûr si c'est le cas, c'est pour toi.

**Reply by elmerdu69** [reply]
Non, on ne pas faire payer des données publiques et puis on peut le considerer comme un service d'intérêt général.
Donc gratuit

**Reply by OUARZA** [reply]
Je m'en occupe ce soir alors.
Je dois me créer une adresse mail spécifique pour le plugin, je le fais ce soir.

**Reply by elmerdu69** [reply]
Salut, je ferai le nettoyage sur la version beta dans le weekend.

**Comment by elmerdu69** [comment]
Salut je viens de faire le nettoyage du code pour la version stable.
Étant donnée que j'ai supprimé des commandes inutiles, je recommande très fortement de 
- supprimer tous les équipements propluvia
- faire la maj
- recréer tous les équipements propluvia

**Comment by OUARZA** [comment]
Hello,
Ce que je te propose c'est de passer en stable quand on basculera sur le market.
Comme ça, les gens feront une installation propre.

**Reply by elmerdu69** [reply]
ça marche !

**Comment by elmerdu69** [comment]
Salut, j'ai vu que tu as validé mon dernier PR. Pense à mettre un message sur le forum pour dire qu'avec cette nouvelle version il est plus que conseillé de recrer les équipements (voir au dessus).

Et sinon au vu du temps qu'il faut pour que tu ais ton compte dev, tu ne penses pas que ça vaudrait quand même la peine de génèrer un stable sur la base de la dernière bêta? 
Dans tous les cas j'ai du taf pour travailler sur les nouvelles idées.

**Comment by OUARZA** [comment]
Hello
1/ oui désolé, je le fais habituellement. J'ai mis à jour le changelog et mis un message sur le forum
2/ Ce que je voulais éviter c'est que les gens l'installent depuis le stable de github et du coup ne passe pas par le market qui permet je crois de savoir combien l'ont installé...
Après si tu me dis de le mettre en stable, je le fais :)

**Reply by elmerdu69** [reply]
ok je comprend, laisse comme c'est. 
Cela n’empêchera pas de faire des updates
